libsplinter/src/admin/store/diesel/mod.rs

```diff
@@ -0,0 +1,43 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+mod models;
+mod schema;
+
+use diesel::r2d2::{ConnectionManager, Pool};
```

Comment:
Looks like the contents of this file aren't in the right commits (should be broken up).